japitana_johnwalter/index.html

```diff
@@ -0,0 +1,200 @@
+          </a>
+        </div>
+      </li>
+    </ul>
+
+    <div class="excercise-7">
+      <div class="name">
+        <h1>japitana john walter</h1>
+      </div>
+      <div>
+        <h2>My interests and hobbies</h2>
+        <ul>
+          <li>Walking</li>
+          <li>Gaming</li>
+          <li>Reading books</li>
+          <li>Listening to music</li>
+          <li>Meditating</li>
+        </ul>
+      </div>
+
+      <div class="works">
+        <h2>My works</h2>
+        <ol>
+          <li><a href="#">Empty1.html</a></li>
+          <li><a href="#">Empty2.html</a></li>
+          <li><a href="#">Empty3.html</a></li>
+          <li><a href="#">Empty4.html</a></li>
+          <li><a href="#">Empty5.html</a></li>
+        </ol>
+      </div>
+
+      <div>
+        <h2>Testimonials</h2>
+        <ul>
+          <li>Amazing as always - akisha</li>
+          <li>Nice webpage! - Jasmin</li>
+          <li>Great Page -James</li>
+        </ul>
+      </div>
+
+      <div class="gallery">
+        <h2>Gallery</h2>
+        <img src="./media/img1.jfif" />
+        <img src="./media/img2.jfif" />
+        <img src="./media/img3.jfif" />
+        <img src="./media/img4.jfif" />
+        <img src="./media/img5.jfif" />
+        <img src="./media/img6.jfif" />
+      </div>
+    </div>
+
+    <div class="centralized">
+      <h2>Contact Me</h2>
+      <form class="container">
+        <div class="item-1">
+          <label>Email: </label>
+          <input type="text" />
+        </div>
+
+        <div class="item-2">
+          <label>Full Name: </label>
+          <input type="text" />
+        </div>
+        <div class="item-3">
+          <label>Age: </label>
+          <input type="text" />
+        </div>
+        <div class="item-4">
+          <label>Subject: </label>
+          <input type="text" />
+        </div>
+
+        <div class="item-5">
+          <label>Send me a copy</label>
+          <input type="checkbox" />
+        </div>
+
+        <div class="item-6">
+          <label>Message</label><br />
+          <textarea></textarea>
+        </div>
+        <div class="item-7">
+          <button type="submit">Submit</button>
+        </div>
+      </form>
+    </div>
+
+    <div class="centralized">
+      <h2>My Goals</h2>
+      <ul>
+        <li>Country-side home</li>
+        <li>My own loving family</li>
+        <li>Have a problem of being rich</li>
+      </ul>
+
+      <form class="container-1">
+        <div class="name-tab">
+          <label>Name: </label>
+          <input type="text" id="text_01" class="user" />
+        </div>
+
+        <div class="comment-tab">
+          <label>Comment</label>
+          <textarea id="textarea_01" class="user-comment"></textarea>
+        </div>
+
+        <div class="button-01">
+          <button id="button_01" type="button" disabled>Comment</button>
+        </div>
+
+        <hr />
+
+        <h2>Comment Section</h2>
+        <select id="sortDropdown">
```

Comment:
IDs should be written in snake case. All lower case separated by underscore.

For reference, follow the coding guidelines.
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR2l29lcF09B6N3jV9JjsQn2kIge3yJWLvkiVWepwKW1vg5YT2a6o6bI2Pw#slide=id.g24da8de92d7_0_5